Travis npm integration error

**Describe the bug**
While deploying the app from [Travis](https://travis-ci.org), the `npm` deploy gets the error `npm ERR! Version not changed, might want --allow-same-version`.

**To Reproduce**
Steps to reproduce the behavior:
1. Create tag.
2. Wait for [Travis](https://travis-ci.org)error.

**Expected behavior**
It should deploy to  [Travis](https://travis-ci.org)

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,12 @@ script:
 before_deploy:
   - npm config set git-tag-version false
   - npm version $TRAVIS_TAG
+  - git commit --ammend -m "Change npm version"
+
+after_deploy:
+  - git config --global user.email "travis@travis-ci.org"
+  - git config --global user.name "Travis CI"
+  - git push --force origin master
 
 deploy:
   - provider: npm

